sphinx/builders/latex/__init__.py

```diff
@@ -411,6 +412,15 @@ def validate_config_values(app: Sphinx, config: Config) -> None:
             logger.warning(msg % (key,))
             config.latex_elements.pop(key)
 
+    # latex_elements['package']
+    packages = config.latex_elements.get('packages', [])
+    for pkginfo in list(packages):
+        if not isinstance(pkginfo, (list, tuple)) or len(pkginfo) != 2:
+            msg = __("latex_elements['packages'] contains should be "
+                     "a pair of package name and options: %r")
```

Comment:
Maybe `contents` in place of `contains`? and then "should be a list of pair of package name and options"

Reply:
aghhhh... its typo...